packages/@uppy/remote-sources/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@uppy/remote-sources",
+  "description": "Uppy plugin that includes all remote sources that Uppy+Companion offer, like Instagram, Google Drive, Dropox, Box, Unsplash, Url etc",
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "types": "types/index.d.ts",
```

Comment:
```suggestion
  "types": "types/index.d.ts",
  "type": "module",
```

Reply:
it was right below: https://github.com/transloadit/uppy/pull/3676/commits/9bf0d6797dde2496a1b6e2adeaa2b47ade0ff351#diff-c7a3ab442e33b30a08b16b084646864fd9367a4fb99c293c0c87faf53ae722d9R21 :thinking: